librz/include/rz_asm.h

```diff
@@ -132,7 +132,7 @@ typedef struct rz_asm_plugin_t {
 	void *user; // user data pointer
 	int bits;
 	int endian;
-	bool (*init)(void *user);
+	bool (*init)(void **user);
```

Comment:
why `**user`?
If you need to return a pointer can't we just do:
```suggestion
	void *(*init)(void *user);
```
?

Reply:
because we want to handle errors.

Reply:
Does this mean that you could have `*user` set but with an error? Or `*user` not set but without errors?

Reply:
the arch might not need to return a data structure.